xml/System.Net/NetworkCredential.xml

```diff
@@ -79,6 +79,11 @@
 ## Remarks
  The <xref:System.Net.NetworkCredential> class is a base class that supplies credentials in password-based authentication schemes such as basic, digest, NTLM, and Kerberos. Classes that implement the <xref:System.Net.ICredentials> interface, such as the <xref:System.Net.CredentialCache> class, return <xref:System.Net.NetworkCredential> objects.
 
+> [!NOTE]
+> On Unix platforms, use of NTLM authentication may require installing the NTLM plugin for the GSSAPI layer used by the implemention. On Ubuntu distributions, the package is called `gss-ntlmssp`. Alternatively, applications may opt in for a managed implementation of NTLM by setting the `_UseManagedNtlm` .csproj property (.NET 9+ only).
```

Comment:
I would still document the AppConext properties (end ENV if any)  as well e.g. leave  it together with `_UseManagedNtlm` . Especially since it does not work in 8.0 that is LTS.